selfdrive/ui/tests/test_translations.py

```diff
@@ -121,6 +123,56 @@ def test_entities_error(self):
         matches = re.findall(r'@(\w+);', cur_translations)
         self.assertEqual(len(matches), 0, f"The string(s) {matches} were found with '@' instead of '&'")
 
+  def test_bad_language(self):
+    """
+    Tests if the given translation files contain any bad language using LDNOOBW.
+    """
+    AVAILABLE_LANGUAGE_CODES = ['ar', 'cs', 'da', 'de', 'en', 'eo', 'es', 'fa', 'fi', 'fil', 'fr', 'hi', 'hu', 'it', 'ja', 'kab', 'ko', 'nl', 'no', 'pl',
+                                'pt', 'ru', 'sv', 'th', 'tlh', 'tr', 'zh']
+    OVERRIDE_WORDS = ['pédale']
+
+    for name, file in self.translation_files.items():
```

Comment:
now that we're adding test cases, we should refactor this class to parameterize the class instead of having a for loop in every test. I can do this later or you can try in a separate PR

Reply:
https://github.com/commaai/openpilot/pull/30842